components/org.wso2.carbon.identity.api.server.configs/org.wso2.carbon.identity.api.server.configs.common/src/main/java/org/wso2/carbon/identity/api/server/configs/common/ConfigsServiceHolder.java

```diff
@@ -97,4 +99,23 @@ public void setCorsManagementService(CORSManagementService corsManagementService
 
         ConfigsServiceHolder.getInstance().corsManagementService = corsManagementService;
     }
+
+    /**
+     * Get RemoteLoggingConfigService osgi service.
+     *
+     * @return RemoteLoggingConfigService
+     */
+    public RemoteLoggingConfigService getRemoteLoggingConfigService() {
+
+        return ConfigsServiceHolder.getInstance().remoteLoggingConfigService;
```

Comment:
shouldn't this be `this.remoteLoggingConfigService` ?

Reply:
I think both are fine, but the suggestion is more appropriate.